week1-section3/exercise 3/Main.cpp

```diff
@@ -0,0 +1,28 @@
+/*Exercise 3 - Multiples of 10
+Write a program that will output multiples of 10 between 1 and 100. Use a for-loop that goes through numbers 1-100.	
+
+At the end of the program, output the sum of all the numbers that were NOT multiples of 10. Do NOT use an else statement for this program and do NOT use more than one loop.
+
+*/
+#include <iostream>
+
+int main() {
+
+	
+	int sum=0;
+	int multiples=0;
+	std::cout << "These the multyples of 100" << std::endl;
+
+	for (int i = 0; i <= 100; i++)
+	{
+		//give the multiples of 10
+
+		if (i % 10 == 0) {
+			std::cout << i << ", ";
+			multiples = multiples + i;
```

Comment:
This is fine and there is no need to change it. But just as a comment, keep in mind that you can also write `multiples += i`